bucket/btop-lhm.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "1.0.2",
+    "description": "btop++ for windows",
+    "homepage": "https://github.com/aristocratos/btop4win",
+    "license": "Apache-2.0",
+    "notes": [
+        "Has full GPU monitoring, etc. using Libre Hardware Monitor, DLL's included in the package.",
+        "Requires admin rights to run."
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/aristocratos/btop4win/releases/download/v1.0.2/btop4win-LHM-x64.zip",
+            "hash": "898b0725918e58926851304195e795b881abd1c2353b0537e62ea1a273777a66",
+            "extract_dir": "btop4win"
+        }
+    },
+    "pre_install": [
+        "Get-ChildItem \"$dir\\btop4win.exe\" | Rename-Item -NewName { $_.name -Replace '4win', '' }",
+        "if (!(Test-Path \"$persist_dir\\btop.conf\")) { New-Item \"$dir\\btop.conf\" | Out-Null }"
+    ],
+    "pre_uninstall": [
+        "$sys = \"$dir\\btop.sys\"",
+        "if (!(Test-Path $sys)) { return }",
+        "$tempPath = '~\\appdata\\temp\\btop.sys'",
+        "if (Test-Path $tempPath) { $i = 0; While (Test-Path $tempPath) { $i++; $tempPath = \"$($tempPath | Split-Path)\\btop$i.sys\" } }",
+        "Move-Item $sys $tempPath"
+    ],
+    "persist": "btop.conf",
+    "bin": "btop.exe",
```

Do not rename the original executable, instead create a shim named `btop`

```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\btop.conf\")) { New-Item \"$dir\\btop.conf\" | Out-Null }",
    "pre_uninstall": [
        "$sys = \"$dir\\btop.sys\"",
        "if (!(Test-Path $sys)) { return }",
        "$tempPath = '~\\appdata\\temp\\btop.sys'",
        "if (Test-Path $tempPath) { $i = 0; While (Test-Path $tempPath) { $i++; $tempPath = \"$($tempPath | Split-Path)\\btop$i.sys\" } }",
        "Move-Item $sys $tempPath"
    ],
    "persist": "btop.conf",
    "bin": [
        "btop4win.exe",
        [
            "btop4win.exe",
            "btop"
        ]
    ],
```